internal/core/integrations/github_integration.go

```diff
@@ -663,9 +668,13 @@ func (g *githubIntegration) CreateIssue(ctx context.Context, asset models.Asset,
 
 	assetSlug := asset.Slug
 	labels := getLabels(&dependencyVuln, "open")
+	componentTree, err := g.renderPathToComponent(asset.ID, assetVersionName, "SBOM-File-Upload", exp.AffectedComponentName)
```

Comment:
I believe you can get the scanner id from the vulnerability

```suggestion
	componentTree, err := g.renderPathToComponent(asset.ID, assetVersionName, dependencyVuln.ScannerID, exp.AffectedComponentName)
```